desktop/cmp/grid/impl/filter/custom/CustomRowModel.ts

```diff
@@ -82,7 +82,7 @@ export class CustomRowModel extends HoistModel {
         makeObservable(this);
 
         let newOp = op as OperatorOptionValue;
-        if (isNil(value)) {
+        if (isNil(value) || (isArray(value) && isEmpty(value))) {
```

Comment:
might add a util `isEmptyArray()` -- we do this check 4 times in this PR -- might be others in hoist